packages/Webkul/Shop/src/Http/Controllers/ProductsCategoriesProxyController.php

```diff
@@ -48,13 +51,22 @@ public function index(Request $request)
             if ($product = $this->productRepository->findBySlug($slugOrPath)) {
 
                 $customer = auth()->guard('customer')->user();
+                
+                $orderedList = $this->orderItem->getModel()->where('product_id', $product->id)
+                ->leftJoin('orders', 'order_items.order_id', 'orders.id')
+                ->where('orders.customer_id', $customer->id)
+                ->get();
```

Comment:
We can use the count function here instead of get function for getting all the items and then count